src/preprocessor.ts

```diff
@@ -55,14 +55,28 @@ export function process(
     fileName: filePath,
   });
 
+  let tsTranspiledText = tsTranspiled.outputText;
+  if (tsJestConfig.ignoreCoverageForAllDecorators === true) {
+    tsTranspiledText = tsTranspiledText.replace(
+      /__decorate/g,
+      '/* istanbul ignore next */__decorate',
+    );
+  }
+  if (tsJestConfig.ignoreCoverageForDecorators === true) {
+    tsTranspiledText = tsTranspiledText.replace(
+      /(__decorate\(\[\r?\n[^\n\r]*)\/\*\s*istanbul\s*ignore\s*decorator(.*)\*\//g,
```

Comment:
As it's `ts-jest` which is parsing the `/* istanbul ignore decorator */` comments, maybe it should be named `/* ts-jest ignore decorator coverage */`.

Reply:
the comment is actually processed by istanbul and not by ts-jest so the current form is correct